regression/cbmc-java/lvt-unexpected/test.desc

```diff
@@ -10,3 +10,5 @@ expected by the bytecode to goto conversion algorithms. Namely, the live range
 of parameter x in that method is not preceeded by a store_i instruction, which
 is perfectly possibly as per the JVM specs but currently additionally required
 in CBMC.
+
+This doesn't work under lazy loading because no entry-point function is given.
```

Comment:
Probably should call it "symex driven lazy loading" since otherwise will get confusing. et a.